docs/installation/debian.md

```diff
@@ -75,6 +75,11 @@ from the new repository:
 
             deb https://apt.dockerproject.org/repo debian-stretch main
 
+> **Note**: Docker does not provide packages for all architectures. To install docker on
+> a multi-architecture system, add an `[arch=...]` clause to the entry. Refer to the
+> [Debian Multiarch wiki](https://wiki.debian.org/Multiarch/HOWTO#Setting_up_apt_sources)
+> for details.
+
```

Comment:
Ah, my eyes missed that as I was focusing too much on the alignment. Fixed now. Thanks for your patience :).